### PR TITLE
Merge 4.37 to trunk

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -164,20 +149,27 @@ GEM
       xcpretty-travis-formatter (>= 0.0.3)
     fastlane-plugin-appcenter (1.8.0)
     fastlane-plugin-sentry (1.6.0)
+    fastlane-plugin-wpmreleasetoolkit (1.0.1)
```
Comment (AliSoftware) [inline Gemfile.lock:152]: We're at 1.2.0 now, probably worth updating on next cycle (also, did I update SNiOS to call `check_for_toolkit_update` already? I thought I did it in last release branch, maybe I did it in develop instead, worth a quick check for your next cycle.

File: Simplenote/Simplenote-Info.plist [hunk 2]
```diff
@@ -55,6 +53,8 @@
 			</dict>
 		</array>
 	</dict>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
```
Comment (AliSoftware) [inline Simplenote/Simplenote-Info.plist:57]: Nice, one less thing to update manually in AppStoreConnect's UI when submitting 👍

**Review by AliSoftware — APPROVED** [review]
:shipit:

File: Gemfile.lock (hunk 1 above)
Comment (oguzkocer) [reply Gemfile.lock:152]: > did I update SNiOS to call check_for_toolkit_update already

It doesn't look like it: https://github.com/Automattic/simplenote-ios/pulls?q=is%3Apr+author%3AAliSoftware+is%3Aclosed

Comment (AliSoftware) [reply Gemfile.lock:152]: Dammit indeed. Sorry, my bad; I'll let you add it (it's a one-liner in `before_all`) during your next code freeze if you want it.

**Pull request merged**